rosdep/base.yaml

```
@@ -460,6 +460,15 @@ castxml:
   fedora: [castxml]
   macports: [castxml]
   ubuntu: [castxml]
+catch2:
+  arch: [catch2]
+  debian: [catch2]
+  fedora: [catch2-devel]
```

I don't think this is available for Fedora 37.
```suggestion
  fedora:
    '*': [catch2-devel]
    '37': null
```

---

good catch, thank you